Snippets/Snippets_4/Host/EndpointStartAndStop.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Snippets4.Host
+{
+    using NServiceBus;
+
+    #region host-EndpointStartAndStop
```

Comment:
@colin-higgins - why do we need this snippet. We already have this? snippet:lifecycle-EndpointStartAndStop